schemas/app/AccountingSettings.json

```diff
@@ -86,6 +86,13 @@
       "default": false,
       "section": "Features"
     },
+    {
+      "fieldname": "enableStockReturns",
```

Comment:
Why is this here? it should be in InventorySettings, it's a feature under Inventory.

Reply:
corrected